README.md

```diff
@@ -104,99 +106,53 @@ npm run build && node ./build/src/index.js
 zk file <name>  # or path/to/name
 ```
 
-This will create `name.js` and `name.test.js`.
+This command creates `name.js` and `name.test.js`.
 
-For convenience, running this command in your project's _root_ directory will
-create the files inside your project's `src` dir automatically, even if you
-don't specify `src/` as part of your file path. When _not_ in your project's
-root dir, files will be created at the path you specify relative to your
-terminal's current working directory.
+For convenience, when you run this command in your project's _root_ directory creates the files inside your project's `src` dir automatically, even if you don't specify `src/` as part of your file path. 
+
+When you run this command when you are _not_ in your project's
+root dir, files are created at the path you specify relative to your terminal's current working directory.
 
 ### Show system info
 
 ```sh
 zk system
 ```
 
-This will output system info such as your NodeJS version, NPM version,
-`zkapp-cli` version, etc. Please include this if submitting a bug report for
-`zkapp-cli`, for easier troubleshooting.
+This command outputs system info such as your NodeJS version, NPM version, `zkapp-cli` version, and so on. For easier troubleshooting, be sure to include this information if submitting a bug report for `zkapp-cli`. See the [CONTRIBUTING](https://github.com/o1-labs/zkapp-cli/blob/main/README.md) guidelines.
 
 ### Update your config.json
 
 ```sh
 zk config
 ```
 
-`config.json` is an auto-generated file and contains your deployment-related
-configurations.
+The auto-generated `config.json` file contains your deployment-related configurations.
 
 Run the `zk config` command to add a new deployment configuration to this file.
 
-This command is interactive and will prompt you to specify 1.) a network name
-(can be anything, e.g. `testnet`), 2.) the Mina GraphQL API URL where you want to
-send your deployment transaction (e.g.
-`https://proxy.berkeley.minaexplorer.com/graphql`), and 3.) the transaction fee
-to be used during deployment (in MINA; e.g. `0.01`). The URL is significant
-because this determines which network you're deploying to (e.g. `QANet`,
-`Testnet`, etc).
-
-Note: If your project contains more than one smart contract that you will be
-deploying, we recommend following an alias naming convention such as `testnet-foo`
-and `testnet-bar`, where `testnet` is the name of the network and `foo` and `bar`
-are the name of your smart contracts. Then you can deploy using `zk deploy testnet-foo` and `zk deploy testnet-bar`.
+This interactive command is interactive prompts you to specify a name (can be anything), URL to deploy to, and fee (in
```

Comment:
This sentence reads a little weird to me.

Suggestion:

"this interactive command is interactive prompts you" -> this interactive command prompts you"

Reply:
We should also mention the fee payer config item recently added.